cross-dom-comm/hardhat/.env.example

```diff
@@ -1,3 +1,5 @@
-MNEMONIC= << your value here >>
-GOERLI_URL= << your value here >>
-OPTI_GOERLI_URL= << your value here >>
+# Put the mnemonic for an account on Optimism here
+MNEMONIC="test test test test test test test test test test test junk"
+GOERLI_KEY=  <<value goes here>>
+OPTIMISM_GOERLI_KEY= <<value goes here>>
```

Comment:
Should this key not be called 
OPTIMISM_GOERLI_ALCHEMY_KEY
So that it is very explicit and less confusing as to what it is?

Reply:
Need to do that in all the tutorials. Working on it now.